docs/custom.css

```
@@ -1195,10 +1186,39 @@ article p {
+    padding-inline-end: var(--sidebar-caret-inset);
+  }
+
+  .theme-doc-sidebar-container .menu__caret {
+    padding-inline-end: var(--sidebar-caret-inset);
+  }
+
   #doc-item-container {
     gap: 2rem;
   }
 
+  .menu__link {
```

Stylelint (17.14.0)

[error] Expected class selector ".menu__link" to be kebab-case (selector-class-pattern)

(selector-class-pattern)